source/slang/diff.meta.slang

```diff
@@ -151,3 +151,62 @@ struct __DifferentialPair
         return p();
     }
 };
+
+typealias IDFloat = IFloat & IDifferentiable;
+
+namespace dstd
```

Comment:
We should make a decision to either define these on the original stdlib functions or duplicate them in dstd namespace.

Reply:
One reason why I couldn't place the decorations on the original declarations in hlsl.meta.slang is that those are defined with __BuiltinArithmeticType. This is fine for the intrinsic op since it gets handled internally, but the differential function has a body and cannot do much with __BuiltinArithmeticType.

I think once those functions are moved to IFloat, we could think about declaring IFloat as extending IDifferentiable, and move the modifiers on to the original declarations.